app/lib/common/widgets/redact_content.dart

```diff
@@ -113,30 +107,19 @@ class RedactContentWidget extends ConsumerWidget {
         );
       }
 
-      if (ctx.mounted) {
-        Navigator.of(ctx, rootNavigator: true).pop();
-        Navigator.of(ctx, rootNavigator: true).pop(true);
-        customMsgSnackbar(ctx, L10n.of(ctx).contentSuccessfullyRemoved);
-        if (onSuccess != null) {
-          onSuccess!();
-        }
+      EasyLoading.dismiss();
+      if (!ctx.mounted) return;
+      EasyLoading.showToast(L10n.of(ctx).contentSuccessfullyRemoved);
```

Comment:
`showSuccess` is a very obstructive UI call: with a centered entry that takes a lot of attention. Unless we need the user to wait until that is gone (because we are loading something specific in the background or it takes a moment for it to take effect), I told @bitfriend to use `showToast` for most cases to have a less attention-grabbing response. Like in this case, where there is no reason to be that in-the-face about it.